Workshops/2023-06/README.md

```diff
@@ -0,0 +1,4 @@
+# ESMValTool workshop: June 2023
+
+See https://github.com/ESMValGroup/Community/discussions/## for more
```

Comment:
The link seems broken

Reply:
Thanks @bouweandela! The discussion didn't exist when I created this PR, but I have updated it now: f7316f8 :)